templates/sysdig-cloud/0/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+sysdig-agent:
+  container_name: sysdig-agent
+  privileged: true
+  image: sysdig/agent
```

Comment:
I know this is latest, but it would be nice if the user could specify. That way if it matters to them, they can get a consistent release across their environment. Othewise overtime they could get drift. 

Set the default to latest in the questions.